tests/boot/boot_from_pxe.pm

```diff
@@ -222,8 +222,9 @@ sub run {
         save_screenshot;
         # We have textmode installation via ssh and the default vnc installation so far
         if (check_var('VIDEOMODE', 'text') || check_var('VIDEOMODE', 'ssh-x')) {
-            $is_nvdimm && check_var('VIDEOMODE', 'text') ? type_string_very_slow('DISPLAY= ') : type_string_slow('DISPLAY= ');
-            $is_nvdimm ? enter_cmd_very_slow("yast.ssh") : enter_cmd_slow("yast.ssh");
+            #$is_nvdimm && check_var('VIDEOMODE', 'text') ? type_string_very_slow('DISPLAY= ') : type_string_slow('DISPLAY= ');
+            #$is_nvdimm ? enter_cmd_very_slow("yast.ssh") : enter_cmd_slow("yast.ssh");
+            enter_cmd_slow("yast.ssh");
```

Comment:
Temporary change right ?

Reply:
Yes, the last comit `Julie debug: for verify arm tests`.

To workaround to verify the arm test on BJ qe2 openqa server. I'll mark it [WIP] until verification finishes.